cassandra/assets/configuration/spec.yaml

```
@@ -0,0 +1,30 @@
+name: Cassandra
+files:
+- name: cassandra.yaml
+  options:
+  - template: init_config
+    options:
+      - template: init_config/jmx
+  - template: instances
+    options:
+    - template: instances/jmx
+      overrides:
+        host.value.example: localhost
+        port.value.example: 7199
+    - name: cassandra_aliasing
+      description: |
+        Must be set to true to comply with CASSANDRA-4009.
+        Learn more: https://issues.apache.org/jira/browse/CASSANDRA-4009
+      required: true
+      value:
+        type: boolean
+        example: true
```

```suggestion
    options:
    - name: cassandra_aliasing
      description: |
        Must be set to true to comply with CASSANDRA-4009.
        Learn more: https://issues.apache.org/jira/browse/CASSANDRA-4009
      required: true
      value:
        type: boolean
        example: true
    - template: instances/jmx
      overrides:
        host.value.example: localhost
        host.display_priority: 1
        port.value.example: 7199
        port.display_priority: 1
```
Super small nit: we can bump the `cassandra_aliasing` option higher up if the config option gets moved before the template and then bumping the display priority of `host` and `port`.